libraries/AP_HAL_ChibiOS/hwdef/Sierra-F1/README.md

```diff
@@ -0,0 +1,74 @@
+# Sierra-F1 Autopilot by Sierra Aerospace
```

Comment:
Need a link to where this can be bought

Reply:
Website is in making. Rest details will be updated as soon as we finish testing. Still the testing is going on.
Thank you!